CHANGELOG.md

```diff
@@ -47,6 +48,7 @@
 
 ## Breaking changes
 
+- Updated solid-phase diffusivity functions in `Ecker2015` and `Ecker2015_graphite_halfcell` parameter sets to better match the original papers ([#4141](https://github.com/pybamm-team/PyBaMM/pull/4141))
```

Comment:
This should probably be under only "breaking changes" or "features" instead of both